tools/react-app-template/package.json

```diff
@@ -0,0 +1,122 @@
+{
+    "name": "@gooddata/react-app-template",
+    "version": "1.0.0",
+    "description": "GoodData Template for React App",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/gooddata/gooddata-ui-sdk.git",
+        "directory": "tools/react-app-template"
+    },
+    "author": "GoodData Corporation",
+    "private": true,
+    "license": "MIT",
+    "main": "dist/index.js",
+    "typings": "dist/index.d.ts",
+    "sideEffects": [
+        "*.css",
+        "*.svg"
+    ],
+    "files": [
+        "dist/**/*.js",
+        "dist/**/*.json",
+        "dist/**/*.d.ts",
+        "dist/**/*.map",
+        "dist/**/*.svg"
+    ],
+    "scripts": {
+        "clean": "rm -rf ci dist esm coverage *.log styles/css && jest --clearCache",
+        "build": "webpack --mode production",
+        "start": "webpack-cli serve",
+        "test": "jest --watch --passWithNoTests",
+        "test-once": "jest --maxWorkers=${JEST_MAX_WORKERS:-'45%'} --passWithNoTests",
+        "test-ci": "JEST_JUNIT_OUTPUT=./ci/results/test-results.xml jest --ci --config jest.ci.js --passWithNoTests",
+        "eslint": "eslint -c .eslintrc.js --ext ts,tsx src/",
+        "eslint-ci": "mkdir -p ./ci/results && eslint -f checkstyle -o ci/results/eslint-results.xml -c .eslintrc.js --ext ts,tsx src/",
+        "prettier-check": "prettier --check \"{src,test}/**/*.{ts,tsx,json,scss,md,yaml,html}\"",
+        "prettier-write": "prettier --write \"{src,test}/**/*.{ts,tsx,json,scss,md,yaml,html}\"",
+        "dep-cruiser": "depcruise --validate .dependency-cruiser.js --output-type err-long src/",
+        "dep-cruiser-ci": "depcruise --validate .dependency-cruiser.js --output-type err-long src/",
+        "validate": "npm run dep-cruiser && npm run eslint && npm run prettier-check",
+        "validate-ci": "npm run dep-cruiser-ci && npm run eslint-ci && npm run prettier-check",
+        "refresh-md": "gdc-catalog-export"
+    },
+    "dependencies": {
+        "@gooddata/sdk-backend-base": "^8.12.0-alpha.62",
+        "@gooddata/sdk-backend-bear": "^8.12.0-alpha.62",
+        "@gooddata/sdk-backend-spi": "^8.12.0-alpha.62",
+        "@gooddata/sdk-backend-tiger": "^8.12.0-alpha.62",
+        "@gooddata/sdk-model": "^8.12.0-alpha.62",
+        "@gooddata/sdk-ui": "^8.12.0-alpha.62",
+        "@gooddata/sdk-ui-charts": "^8.12.0-alpha.62",
+        "@gooddata/sdk-ui-dashboard": "^8.12.0-alpha.62",
+        "@gooddata/sdk-ui-ext": "^8.12.0-alpha.62",
+        "@gooddata/sdk-ui-filters": "^8.12.0-alpha.62",
+        "@gooddata/sdk-ui-geo": "^8.12.0-alpha.62",
+        "@gooddata/sdk-ui-pivot": "^8.12.0-alpha.62",
+        "json-stable-stringify": "^1.0.1",
+        "lodash": "^4.17.19",
+        "tslib": "^2.0.0"
+    },
+    "peerDependencies": {
+        "react": "^16.10.0 || ^17.0.0",
```

Comment:
If this is a final app template, I think the React and React DOM should be normal dependencies.

Reply:
Good catch! Thanks :)